docs/learnpaths/paths.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: Introduction
+sidebar_position: 10
+title: Introduction
+description: Learning paths for different personas.
+keywords: [learning paths]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [learning-paths]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/learning-paths/paths"/>
+</head>
+
+If you are new to Kubewarden, these sections contain a set of learning paths for different personas.
+
+|Persona|Description|
+|-|-|
+|[user](kubewarden-user.md)|Someone who takes a policy and uses it in a cluster. They run policies and observe results.|
```

Comment:
Maybe better "they deploy policies" instead of "run policies".

Reply:
Yes, that's more accurate. Thanks.